### DISCUSSION TITLE
specify custom data when exporting the song playback timeline

### DISCUSSION THREAD

**Discussion by Aerobird98** [AzuraCast/AzuraCast#4378] [Q&A] [answered]
Hi all,

I have a question.

Is it possible to monitor and save some more specific informations like the song's title, its author, the lenght of the song, the song's label (I mean its publisher) and the time when it is first published when exporting the song playback timeline?

Currently when I attempt to export it, it always contains the same informations (date/time, etc) that is good but I also need the informations listed above. Is it even possible to do it right now?
![screenshot_of_the_song_playback_timeline_2021-07-05 103551](https://user-images.githubusercontent.com/19577295/124442498-18b38200-dd7d-11eb-8aab-c7724a10c0f0.jpg)
I want the exact same informations found inside the filtering options on the music files tab.
![Képernyőkép 2021-07-05 103920](https://user-images.githubusercontent.com/19577295/124443102-b018d500-dd7d-11eb-8597-d1a051603848.jpg)

Thanks in advance and sorry for the hungarian screenshots,
Aerobird98

**Comment by BusterNeece** [comment, marked as answer]
@Aerobird98 At the moment it's not possible to customize what data is included in that report, though that is an excellent candidate for a new feature for us to work on in the future.

I'd recommend requesting it as a new feature over on https://features.azuracast.com/ so the community can vote on it as a priority for our development efforts.

**Reply by Aerobird98** [reply]
Thanks! I will make a suggestion then.

Regards,
Aerobird98
